index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my portfolio</title>
+    <link rel="stylesheet" href="styling.css">
+    <link href="Inter-VariableFont_slnt,wght.ttf" rel="stylesheet">
+    <link href="CreteRound-Regular.ttf" rel="stylesheet">
+    <link href="Roboto-Regular.ttf" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+          <nav class="navigation">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+          </nav>
```

Comment:
- [ ] I suggest that you use the ``header`` tag rather than a ``div``, because its a good practice to structure your HTML accordingly this also helps it become more responsive 👍